src/Arcus.Observability.Tests.Unit/Serilog/ApplicationInsightsTelemetryConverterTests.cs

```diff
@@ -0,0 +1,353 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using Arcus.Observability.Telemetry.Core;
+using Arcus.Observability.Telemetry.Serilog.Sinks.ApplicationInsights;
+using Arcus.Observability.Telemetry.Serilog.Sinks.ApplicationInsights.Converters;
+using Bogus;
+using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.DataContracts;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Serilog;
+using Serilog.Context;
+using Serilog.Core;
+using Serilog.Events;
+using Serilog.Extensions.Logging;
+using Serilog.Formatting.Display;
+using Xunit;
+using static Arcus.Observability.Telemetry.Core.ContextProperties;
+using HttpMethod = Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpMethod;
+using ILogger = Microsoft.Extensions.Logging.ILogger;
+
+namespace Arcus.Observability.Tests.Unit.Serilog
+{
+    [Trait("Category", "Unit")]
+    public class ApplicationInsightsTelemetryConverterTests
```

Comment:
Did my best to not make the tests to abstract or "reusable" for beginners to hard to read. So, tried some examples instead bc the other `ILoggerExtensionsTests` are testing more thoroughly the writting itself while this tests the conversion to telemetry.